### DISCUSSION TITLE
server-side Newsletter Subscription

### DISCUSSION THREAD

**Discussion by cupidanish10** [apostrophecms/apostrophe#2923] [Q&A]
Hello, I'm frontend developer and i want to add server-side Newsletter Subscription to my website. So how can I achieve that from apostrophe cms. Never done it before. or is there any other way? like other plugins / tutorials. 
Thank you!

**Comment by abea** [comment]
@cupidanish10 What features are you looking to add for subscription? There is [Apostrophe Forms](https://www.npmjs.com/package/apostrophe-forms), which you could use to add a form to the page to collect subscriptions. You can choose a few ways to get those submissions back to you, including [a Google Sheets module](https://www.npmjs.com/package/apostrophe-forms-submit-google).

There's a documentation tutorial page all about creating forms, including more hand-coded techniques if you don't need flexible form creation: https://docs.apostrophecms.org/howtos/forms.html
